enterprise/api/v1/views.py

```diff
@@ -385,9 +385,6 @@ def get_serializer_class(self):
 
     @method_decorator(require_at_least_one_query_parameter('course_run_ids', 'program_uuids'))
     @detail_route()
-    @permission_required(
```

Comment:
@brittneyexline So we will be add this permission check back on before rolling out Authz changes ?

Reply:
NVM, just read the description part regarding this.